krm/1_clusters/cluster-dev.yaml

```diff
@@ -0,0 +1,122 @@
+apiVersion: container.cnrm.cloud.google.com/v1beta1
+kind: ContainerCluster
+metadata:
+  name: dev-us-central1
+  namespace: YOUR_NAMESPACE # {"$kpt-set":"namespace"}
+  annotations:
+    cnrm.cloud.google.com/remove-default-node-pool: "true" # don't create the default node pool
+spec:
+  initialNodeCount: 1 # this field must be set to greater than 0
+  addonsConfig:
+    horizontalPodAutoscaling:
+      disabled: false
+    httpLoadBalancing:
+      disabled: false
+    istioConfig:
+      disabled: true
+    networkPolicyConfig:
+      disabled: false
+  clusterAutoscaling:
+    enabled: false
+  databaseEncryption:
+    state: DECRYPTED
+  defaultMaxPodsPerNode: 110
+  enableBinaryAuthorization: true
+  ipAllocationPolicy:
+    clusterSecondaryRangeName: anthos-platform-pods-dev
+    servicesSecondaryRangeName: anthos-platform-services-dev
+  location: us-central1
```

Comment:
This should probably be a setter.

Reply:
Should this be a setter per environment? e.g. 4 setters: dev-location, staging-location, prod-location and prod-location

Or per location? e.g. 2 setters: first-region (`us-central1`, which is used by both dev, staging, and prod 1) and second-region (`us-east1`)

Reply:
For now, I think it should be a setter per env.

Hopefully we can come up with a better solution from the DRY config brainstorm going forward. I'm not very happy with the options available now.

Reply:
Clusters also references subnetworks whose name implies their location (e.g. `anthos-platform-central`). It'd be tricky to make the location setter here match the subnetwork name, especially when subnetworks are created in the foundation layer. 

We need a better way to solve this too.

Reply:
Agreed, we'll need a better solution for that.